src/styles/_catalog.scss

```diff
@@ -1,14 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1,200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2,200px);
+  }
+
+  @media (min-width:768px) {
+    grid-template-columns: repeat(3,200px);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4,200px);
+  }
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
-    align-items: center;
+    align-content: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
```

Comment:
In the cheklist they ask us not to forget to add the hover properties from our previous task, I would recommend you to add it in the header and also in the cards.